src/main/java/entities/Post.java

```diff
@@ -9,33 +9,33 @@
  * description,tags,collaborators, deadline, and users who favourited it.
  */
 public class Post extends Postable{
-    public static int numPostsCreated = 0;
-    /**The id is equal to the number of posts created.*/
-    private int id;
+
     /**This is a list of the users who favourited this post.*/
     private List<User> favouritedUsers;
```

Comment:
should be List<Integer> or ArrayList<Integer>

Reply:
Also containing Integers